examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,558 @@
+"""
+==============================
+Plotting Novel Tract Profiles:
+==============================
+
+The following is an example of tractometry for a novel bundle and plotting the
+resulting FA tract profile. We will run tractometry for the *anterior forceps*
+using waypoint ROIs.
+
+**AFQ Waypoint ROI Tractometry:**
+
+.. note::
+
+  This example uses the Yeatman et al. waypoint ROI approach, first
+  described in [Yeatman2012]_ and further elaborated in [Yeatman2014]_.
+
+  The waypoint ROIs used in this example are from the anterior frontal lobe of
+  the corpus callosum (AntFrontal). The waypoint ROIs are from the human corpus
+  callosum templates:
+
+    https://figshare.com/articles/Templates_for_Automated_Fiber_Quantification_of_corpus_callosum_from_Diffusion_MRI_data/3381523
+
+"""
+import os.path as op
+import matplotlib.pyplot as plt
+import numpy as np
+import nibabel as nib
+import dipy.data as dpd
+from dipy.data import fetcher
+from dipy.io.streamline import save_tractogram, load_tractogram
+from dipy.stats.analysis import afq_profile, gaussian_weights
+from dipy.io.stateful_tractogram import StatefulTractogram
+from dipy.io.stateful_tractogram import Space
+
+import AFQ.data as afd
+import AFQ.tractography as aft
+import AFQ.registration as reg
+import AFQ.models.dti as dti
+import AFQ.segmentation as seg
+from AFQ.utils.volume import patch_up_roi
+
+import logging
+import sys
+
+# Ensure segmentation logging information is included in this example's output
+root = logging.getLogger()
+root.setLevel(logging.ERROR)
+logging.getLogger('AFQ.Segmentation').setLevel(logging.INFO)
+logging.getLogger('AFQ.tractography').setLevel(logging.INFO)
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.INFO)
+root.addHandler(handler)
+
+# Monkey patch the progress bars so don't pollute this example's output
+import AFQ._fixes as fixes
+
+def notqdm(iterable, *args, **kwargs):
```

Comment:
I wonder if there's a more elegant way to set up the sphinx config to do this for us, instead of very explicitly doing it here in the example. 

I imagine that if you are running this as a user (say, from the command line) you do want to see the progbar. It's only in the sphinx build context that we don't want it.

Reply:
Once #492 has been merged I will remove the monkey patch